src/constants.rs

```diff
@@ -125,7 +125,7 @@ js_deserializable!(ReturnCode);
 /// ```no_run
 /// use screeps::{find, game, Room};
 ///
-/// let room: Room = game::rooms::get("E23S55").unwrap();
+/// let room: Room = game::rooms::get("E23S55".parse().unwrap()).unwrap();
```

Comment:
There's a missing `)` before the `.parse()`.

Reply:
I believe the current code is correct? We want to parse the string `"E23S55"`, and then send the result of that parse/unwrap operation into `rooms::get`.

Reply:
Oh... I'm the one who missed the other parens. Nevermind me!